ReviewMonitor/AppDelegate.swift

```diff
@@ -53,6 +81,47 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         blurEffectView?.removeFromSuperview()
     }
 
+    var startBack = Date()
+    func applicationDidEnterBackground(_ application: UIApplication) {
+        print(Date())
+        startBack = Date()
+        addBlueView()
```

Comment:
You are adding `blueView` every time application goes to background. This should not be the case. Let's say a user opens the app from background and then they are asked for bio auth. If they "Cancel" the bio auth and puts the app to background and then brings the app tot foreground we are going to have multiple blur background in the app. In this case the user will be stuck in the blur view.

Reply:
Indeed, thanks!

Reply:
I will take care of this one and then merge the pr.